src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+
+#### Example: Custom Slack integration
+
+For example, a GlueScript can perform a custom Slack integration as follows:
+
+```javascript
+var message = activity.text;
+var color;
+
+if (activity.state == 'complete') {
+  if (activity.result == 'success') {
+    color = '#66c000';
+  } else {
+    color = '#ff0000';
+  }
+}
+
+var resp = fetch(
+  '<SLACK_PAYLOAD_URL>',
+  {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      'attachments': [{
+        'fallback': message,
+        'color': color,
+        'fields': [{
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+You can save the file to `.platform/scripts/my_glue_script.js` for example.
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/my_glue_script.js
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> Pushing the script to master won't do anything. Yet. Just a handy place to keep your GlueScript file.
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/my_glue_script.js
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> **note**
+> This is the actual activation of the integration, the script is posted to the API through the CLI. Note that changing the file in Git will not change the integration. To update the integration you will need to use the CLI again.
+>
+> The GlueScript integration can only be managed by the project admin.
+
+# GlueScript APIs
+
+These are the internal Platform.sh APIs that are accessible through GlueScript, API calls are executed with the permissions of the user that created the integration.
+
+## Project API
+
+The project API gives you everything you need to know about the project in the context of which the GlueScript is running. 
+
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
+
+Sub-objects can be accessed directly on the object, like:
+
+```javascript
+project.settings
+```
+
+### Collections
```

Comment:
This section is similar to the above section in that we're drawing an arbitrary distinction between types of properties. Although in this case, we're taking it a step further by describing JavaScript itself; I feel it may be unnecessary to state that an object type that inherently supports iteration does actually support iteration.